binders/kafka-binder/spring-cloud-stream-binder-kafka-reactive/src/main/java/org/springframework/cloud/stream/binder/reactorkafka/ReactorKafkaBinder.java

```diff
@@ -292,8 +251,19 @@ protected MessageProducer createConsumerEndpoint(ConsumerDestination destination
 		DefaultKafkaConsumerFactory<Object, Object> factory = new DefaultKafkaConsumerFactory<>(props);
 		int partitionCount = properties.getInstanceCount() * properties.getConcurrency();
 		boolean groupManagement = properties.getExtension().isAutoRebalanceEnabled();
-		processTopic(consumerGroup, properties, factory, partitionCount, properties.getExtension().isDestinationIsPattern(),
-			groupManagement, destination.getName());
```

Comment:
@sobychacko Actually, this is the main point I want to fix. All the remaining changes I made are for refactoring.